aiohttp/web_log.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import os
 import re
+import time as _time
```

Comment:
I don't think that you need to use an alias here. Please follow the style of the surrounding code.

Reply:
it's needed. the time will be that argument if no alias

Reply:
```suggestion
from time import timezone
```